source/sync/go-to-production/optimize-sync-atlas-usage.txt

```diff
@@ -204,11 +204,26 @@ which collections.
 .. example::
 
    Your app may contain 20 or 30 collections, but you want to minimize the
-   number of queryable fields. You can re-use queryable fields across
-   collections in order to sync objects from every collection. For example,
-   "owner_id" might be a field you want to query in multiple collections, but by
-   having the same field name across collections, only one queryable field is
-   used.
+   number of queryable fields. You can re-use **global queryable fields** 
+   across collections in order to sync objects from every collection. 
+   For example, ``owner_id`` might be a field you want to query in multiple 
+   collections. By making this a **global queryable field** across 
+   collections, only one queryable field is used.
+
+   Alternately, you may have ``owner_id`` in multiple collections, but only
+   need to query on it in one collection. In this case, you might make 
+   ``owner_id`` a **collection queryable field**. This means Sync only has to
+   maintain metadata about this field for one collection, instead of storing
+   metadata for all of the collections where you're not querying on this 
+   field.
+
+   Finally, for simple equality queries, such as ``owner_id == user.id``,
```

Comment:
I think the simple equality note here is not entirely accurate. 

The use case that IQFs make sense in is the same use case as partition based sync where devices want to query on one partition of their data 

Ex. they only care about data for a small subset of (or just one) store/user/etc.

Reply:
Gotcha. I've rewritten this section a bit to be more explicit about the use case. Let me know if this needs further refinement!